.gitignore

```diff
@@ -64,7 +64,7 @@ instance/
 .scrapy
 
 # Sphinx documentation
-docs/_build/
+docs/ _build /
```

Comment:
happy to take the first change, but does this change still pick up that path? Would you reverting before merge? Thank you for this!

Reply:
I do not get your point can you please elaborate as I am new to this. I will be grateful if you will help me out by explaining why codecov rejected this pull request.